app/controllers/figures_controller.rb

```diff
@@ -1,8 +1,11 @@
 # frozen_string_literal: true
 class FiguresController < ApplicationController
-
   def index
-    @figures = Figure.all.order(updated_at: :desc)
+    @figures = if params[:search]
```

Comment:
could pull this out into a private `get_figures` method that you set `@figures` equal to. This works, it would just be a little cleaner